src/material-experimental/mdc-list/_list-theme.scss

```diff
@@ -1,10 +1,33 @@
 @import '@material/list/mixins.import';
+@import '@material/ripple/variables.import';
 @import '../mdc-helpers/mdc-helpers';
 
+// TODO: implement mat-list[dense] once density system is in master
+
 @mixin mat-mdc-list-theme($theme) {
+  $is-dark-theme: map-get($theme, is-dark);
+  $state-opacities:
+      if($is-dark-theme, $mdc-ripple-light-ink-opacities, $mdc-ripple-dark-ink-opacities);
+
   @include mat-using-mdc-theme($theme) {
     @include mdc-list-without-ripple($query: $mat-theme-styles-query);
   }
+
+  // MDC's state styles are tied in with their ripple. Since we don't use the MDC ripple, we need to
+  // add the hover and focus states manually.
+  .mat-mdc-list-item-interactive {
+    &::before {
+      background: if($is-dark-theme, #fff, #000);
+    }
+
+    &:hover::before {
```

Comment:
This and the `focus` style below don't need to be in the theme file.

Reply:
The `$state-opacities` does actually depend on the theme